src/layers.py

```diff
@@ -41,7 +41,14 @@ def __init__(
         super().__init__()
 
         multiple_of = config.get("inner_size_multiple_of", 64)
-        self.act = F.silu
+        self.act_type = config.get("mlp_activation", "gelu")
```

Comment:
Reviewing this now, should we actually default to "silu" given that silu is the current default activation? @Zymrael @exnx

Reply:
wait a min, it's supposed to be gelu, at least that's what the ckpts all use.  This is a source of error accumulation during inference I remember that.  I thought I changed it?

Reply:
@exnx it's changed in other repos. not in `togethercomputer/stripedhyena` though.

Reply:
ah right

Reply:
let's default to silu

Reply:
Done